paddle/framework/framework.proto

```diff
@@ -40,8 +40,8 @@ message OpDesc {
   };
 
   message Var {
-    required string name;                 // e.g. "X"
-    optional int dup = 2 [ default = 0 ]; // e.g., "1"
+    required string op_proto_name = 1;
+    repeated string var_names = 2;
```

Comment:
How about `local_names` means Operator-local names?

I am not sure that I understand these two fields; we need some comments here.